static/js/script.js

```diff
@@ -12,6 +12,20 @@ $(function() {
     $('a[href="' + window.location.hash + '"][data-toggle="tab"]').tab('show');
   }
 
+  // Update location hash after a tab is selected.
```

Comment:
Strictly speaking this JavaScript code doesn't need to run on every page, just on the one where we have the package view. But it's fine with me.